lib/govuk_index/publishing_event_worker.rb

```diff
@@ -60,6 +61,7 @@ def process_action(processor, routing_key, payload)
         processor.save(presenter)
       else
         logger.info("#{routing_key} -> UNKNOWN #{identifier}")
+        GovukError.notify(InvalidFormatError.new, extra: { error_message: "#{presenter.format} is an invalid format" })
```

Comment:
would be good to get the identifier this is for in the error data. Also remember we plan to move to batch process at which point this record will be skipped so we ensure we think about how it would be sent for reprocessing if required. 

That can be as easy as having it resent from publishing API...